### PR TITLE
PMM-12940 Run watchtower with AMI.

### PR DESCRIPTION
**Pull request by BupycHuk**
PMM-12940
PMM-13053
PMM-13063

Link to the Feature Build: SUBMODULES-3664

If this PR adds or removes or alters one or more API endpoints, please review and add or update the relevant [API documents](https://github.com/percona/pmm/tree/main/docs/api) as well:

- [ ] API Docs updated

If this PR is related to some other PRs in this or other repositories, please provide links to those PRs:

- Links to related pull requests (optional).

### CODE REVIEW

**Review by JiriCtvrtka — APPROVED** [review]

File: build/Makefile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 export PACKER_CACHE_DIR := .cache
 export PACKER_VERSION := 1.9.4
+export PMM_SERVER_IMAGE ?= docker.io/perconalab/pmm-server-fb:PR-3664-e536f77
```
Comment (idoqo) [inline build/Makefile:3]: Is the image name a left over? Or we want to keep it until FB is tested?

File: build/ansible/roles/init-admin-password-ami/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+- name: Fetch instance metadata
+  uri:
+    url: http://169.254.169.254/latest/meta-data/instance-id
```
Comment (idoqo) [inline build/ansible/roles/init-admin-password-ami/tasks/main.yml:3]: Wonder if the URL here is meant to be fixed too.

File: build/packer/ansible/roles/cloud-node/files/show-url [hunk 3]
```diff
@@ -5,8 +5,7 @@ SOURCE=
 
 if [ -f /var/lib/cloud/data/status.json ]; then
     SOURCE=$(
-        cat /var/lib/cloud/data/status.json 2>/dev/null \
-            | python -c 'import json, sys; print json.load(sys.stdin)["v1"]["datasource"];' 2>/dev/null
+         cat /var/lib/cloud/data/status.json 2>/dev/null | jq -r '.v1.datasource'  2>/dev/null
```
Comment (idoqo) [inline build/packer/ansible/roles/cloud-node/files/show-url:8]: 👍

**Review by idoqo — APPROVED** [review]

File: build/Makefile (hunk 1 above)
Comment (BupycHuk) [reply build/Makefile:3]: Yes, should be replaced with `perconalab/pmm-server:3-dev-latest` before merge

File: build/ansible/roles/init-admin-password-ami/tasks/main.yml (hunk 2 above)
Comment (BupycHuk) [reply build/ansible/roles/init-admin-password-ami/tasks/main.yml:3]: No, URL is correct, it's internal URL used in AWS to get metadata.

**Pull request merged**